.gitignore

```diff
@@ -17,6 +17,7 @@ utils/tkey_exchange/*.o
 utils/tkey_exchange/sgx_tkey_exchange_t.*
 utils/tkey_exchange/sgx_tkey_exchange_u.*
 utils/ukey_exchange/*.o
+utils/sgx_socket/tls/*.o
 
```

Comment:
update the commit title:
commit msg 改一下：
"
Add support for ra-tls to build the remote secure channel

Previously, the remote secure channel is build with the SIGMA-liked protocol
based on the the libkey_exchange library which has too much coupling codes
and the function extension is also complicated.
In order to decoupling the code implementation, we will change to use the ra-tls
to setup the remote secure channel.

This library is provided by the sgx-sdk and it will embed the quote into the
TLS x.509 certificate as the extension. 
The attestation is mutual, after the quote is verified by both side, a secure
TLS channel will be established between dkeyserver and dkeycache services.

passed all the unit test.
Signed-off-by: <use your intel email>
"

Reply:
sign off 记得改成你的intel 邮箱

Reply:
修改好了